tools/topology/topology2/include/components/asrc_format_s32_to_sxx_convert.conf

```diff
@@ -0,0 +1,346 @@
+#src format array
+			num_input_audio_formats 14
+
+			Object.Base.input_audio_format [
+				# 8khz input
+				{
```

Comment:
These could use the new array definition in alsatplg. Can these be shared, same include files, with SRC?